extensions/aws/tests/PutS3ObjectTests.cpp

```diff
@@ -71,7 +87,7 @@ TEST_CASE_METHOD(PutS3ObjectTestsFixture, "Test AWS credential setting", "[awsCr
     setCredentialsService();
   }
 
-  test_controller.runSession(plan, true);
+  test_controller.runSession(plan);
```

Comment:
Does clang-tidy not run on this file?  It usually complains about "static method accessed through instance" when it sees code like this.

Reply:
I checked and in the clang job `extensions/aws/tests/PutS3ObjectTests.cpp` file was listed as one of the parameters of clang-tidy so it should have run on that file. I'm not sure why it hasn't complained about this.

Reply:
that's weird, but not important